data/json/itemgroups/ammo.json

```diff
@@ -147,7 +147,16 @@
     "type": "item_group",
     "id": "ammo_rifle_rare",
     "//": "Less common rifle ammo including that only used by police/paramilitary forces.",
-    "items": [ [ "300_winmag", 40 ], [ "3006fmj", 50 ], [ "556", 100 ], [ "762_51", 50 ], [ "762_m87", 25 ], [ "8mm_civilian", 15 ] ]
+    "items": [
+      [ "300_winmag", 40 ],
+      [ "3006fmj", 50 ],
+      [ "556", 100 ],
+      [ "762_51", 50 ],
+      [ "4570_sp", 40 ],
```

Comment:
ammo_rifle_rare isn't appropriate for .45-70 ammo. Put it in ammo_obscure.

Reply:
It's no more obscure than .300 Win Mag as a hunting round.

Either way, in practice, the ammo rarely spawns in gun stores and such, likely due to the system's bias towards spawning pistol ammo instead.  I tested it out by spawning 10 gun stores, with 10x item spawn.  I found .45-70 ammo on the shelf exactly once.